src/components/code/utils.tsx

```diff
@@ -27,6 +28,27 @@ const $euiSizeS = 8;
 const isAstElement = (node: RefractorNode): node is AST.Element =>
   node.hasOwnProperty('type') && node.type === 'element';
 
+// Creates an arroy of numbers from comma-separeated
+// string of numbers or number ranges using `-`
+// (e.g., "1, 3-10, 15")
+const parseLineRanges = (ranges: string) => {
+  const highlights: number[] = [];
+  ranges
+    .replace(/ /g, '')
```

Comment:
[Super optional] Any thoughts on making this `\s` which also handles newlines or tabs, if for some reason that comes up as an edge case? 🤔 
```suggestion
    .replace(/\s/g, '')
```

or alternatively, if we want to be even more extra and replace anything that isn't a number, comma, or hyphen:
```suggestion
    .replace(/[^\d,-]/g, '')
```

if you think that's all overkill and makes the regex too unwieldy, definitely feel free to disregard!